ci-operator/step-registry/baremetalds/assisted/operator/baremetalds-assisted-operator-workflow.yaml

```diff
@@ -0,0 +1,18 @@
+workflow:
+  as: baremetalds-assisted-operator
+  steps:
+    pre:
+    - ref: baremetalds-packet-setup
+    - ref: ipi-install-rbac
+    - ref: baremetalds-devscripts-setup
+    - chain: baremetalds-assisted-operator-lso
+    - ref: baremetalds-assisted-operator-setup
+    post:
+    - ref: baremetalds-devscripts-gather
+    - ref: baremetalds-packet-teardown
+    env:
+      DEVSCRIPTS_CONFIG: |
+        IP_STACK=v4
+        NUM_WORKERS=4
```

Comment:
Do we need 4 workers? Default set up is NUM_WORKERS=2 this should be sufficient for now

Reply:
yes, I think you're right